Makefile

```diff
@@ -171,6 +171,7 @@ oracle-db-setup: oracle-database
 schema: ## Runs db schema migrations. Run this when you have changes to your database schema that you have added as new migrations.
 	bundle exec rake db:migrate db:schema:dump
 	MASTER_PASSWORD=p USER_PASSWORD=p ORACLE_SYSTEM_PASSWORD=threescalepass NLS_LANG='AMERICAN_AMERICA.UTF8' DATABASE_URL="oracle-enhanced://rails:railspass@127.0.0.1:1521/systempdb" bundle exec rake db:migrate db:schema:dump
+	DATABASE_URL=$(POSTGRES_DATABASE_URL) bundle exec rake db:migrate
```

Comment:
We could have default one wdyt?

Reply:
I agree with @hallelujah , it should work without params 😄

Reply:
Agreed!. Just thinking which one can be the default 🤔 . any suggestion?

Reply:
`postgresql://postgres:@localhost:5433/circleci`

Reply:
Just don't know about `circleci` as the dbname.. This relies that the user setup their database naming `circleci`. Maybe `threescale`?. WDYT?

any suggestion @hallelujah ?

Reply:
Well I'd say the same as mysql :) `3scale_system_development`